server/internal/repository/review.go

```diff
@@ -100,3 +100,12 @@ func (r *Repository) UpdateReviewContent(ctx context.Context, reviewID uuid.UUID
 	}
 	return nil
 }
+
+func (r *Repository) GetReviews(ctx context.Context) ([]*Review, error) {
+	reviews := []*Review{}
+	if err := r.db.SelectContext(ctx, &reviews, "SELECT reviews, * FROM images JOIN reviews on images.review_id = reviews.id"); err != nil {
```

Comment:
ここでやりたいのは他のところの`SELECT *`と同じです
ただ今回は`reviews`と`images`という2つのテーブルをまとめて扱っているので、どちらのテーブルから取ってくるのかということで`SELECT reviews.*`という書き方になります
```suggestion
	if err := r.db.SelectContext(ctx, &reviews, "SELECT reviews.* FROM images JOIN reviews on images.review_id = reviews.id"); err != nil {
```